app/forms/provider_interface/reconfirm_deferred_offer_wizard.rb

```diff
@@ -68,15 +68,19 @@ def course_option
     def modified_application_choice
       return unless application_choice
 
-      clone = application_choice.dup
-      clone.id = application_choice.id
```

Comment:
Couldn't see why we had to use dup instead of clone, as we still assign the id to the application choice. It was also getting a lit bit more complex as we would then have to unecessarily do the same to offer conditions and then set the id, so switched to using `clone` instead.

Reply:
I suppose `clone` might make a difference if we start using the original application form data to render the previous pages in the wizard flow. Updating the status of the `clone` also updates the status of the original.

Might need to revisit this when fixing that bug around cancelling/pressing back